src/Utilities/ToolLocationHelper.cs

```diff
@@ -109,7 +109,7 @@ public enum TargetDotNetFrameworkVersion
         /// breaking change. Use 'Latest' if possible, but note the
         /// compatibility implications.
         /// </summary>
-        VersionLatest = Version462,
+        VersionLatest = Version48,
```

Comment:
Everything looks good except this. I'm not sure it should change in a minor update for compatibility reasons. But I'm only saying that as the author of that comment that says not to, not that I remember why not :(.

Reply:
IIRC it's because enum values get baked into calling assemblies and we wanted it to be possible to use `VersionLatest` and compile against the latest MSBuild while still running on an earlier patch. Especially given AzDO/GitHub silent VS version updates that still seems like a good conservative policy.